queries/ruby/injections.scm

```diff
@@ -1 +1,6 @@
 (comment) @comment
+
+(heredoc_body
+ (heredoc_content) @content
+ (heredoc_end) @_lang
+ (#inject! @_lang))
```

Comment:
I'm not a huge fan of having to add this directive in order to trigger a hash lookup for the parser name. Is there a way to make this default behaviour when specifying an `@language` capture?

Reply:
For that this would need to be ported upstream, since neovim does the injection. I'll try to look into that, I don't remember if neovim had access to the parser's configs